src/pages/About/AboutTheExperts/AboutTheExperts.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable max-len */
+/*
+  This file contains the HelpComponent, helper functions, and styles
+  The HelpComponent is a static  help page that has FAQ, how to use, data dictionary, and information sheets
+  RenderContent contains all the text listed in the about section
+  styled using CustomStyles from ../../shared/constants
+*/
+
+import {
+  Box, Button, Typography,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { getExperts } from '../../../shared/constants';
+
+const AboutTheExperts = () => {
+  const [value, setValue] = useState(0);
+
+  const councils = [
```

Comment:
add an option in these objects that says whether getExpertsData should return an array of experts or jsx

Reply:
I added an Array.array() check instead of the case 2 and 5 conditional.

Reply:
i think adding a data type option is more flexible and readable